File: frontend/src/types/Tournament.ts
export type TournamentObject = {
    id:string,
    name : string,
    description : string,
    street : string,
    zip : string,
    city : string,
    startDateTime : string,
    endDateTime : string,
    participants : number,
    tugenyId : string,
}

export type Tournament = TournamentObject

File: backend/src/main/java/com/ecosystem/backend/models/Tournament.java
package com.ecosystem.backend.models;

import org.springframework.data.annotation.Id;

import java.util.List;

public record Tournament(
        @Id String id,
        String name,
        String description,
        String street,
        String zip,
        String city,
        String startDateTime,
        String endDateTime,
        Integer participants,
        String tugenyId) {
}
